charts/SAML/values.yaml

```diff
@@ -0,0 +1,42 @@
+deployment:
+  imagePullSecrets: []
+  image: docker.io/insightfinderinc/saml:latest
+  resources:
+    requests:
+        cpu: 100m
+        memory: 300Mi
+
+ingress:
+  ingressClassName: nginx
+  # Host name for ingress/saml server
+  host:
+  tls:
+    enabled: true
+    secretName: ingress-tls
+
+insightfinder:
+  # The URL of the InsightFinder server
+  serverUrl: https://app.insightfinder.com
+  # The URL of the this SAML server
+  samlUrl:
+  # saml application acts as service provider, it needs key and cert to make requests.
+  # key and cert need to be in same secret, with 'key' and 'cert' as the secret keys
+  spSecretName: sp-secret
```

Comment:
We can remove the sp-secret and put a openssl command during container startup to generate the key and certs:
```
openssl req -newkey rsa:2048 -nodes -keyout sp.key -x509 -days 365 -out sp.crt
```